scx.spec

```diff
@@ -0,0 +1,49 @@
+%global debug_package %{nil}
+
+Name: 		scx
+Version:	1%{?dist}
+Release:        %autorelease
+Summary:	sched-ext/scx scheds
+
+
+License:	GPLv2
+URL:		https://github.com/likewhatevs/scx
+Source0:	${nil}
+
+BuildRequires: cargo, rust, elfutils-devel, clang
+
+%description
+scx schedulers packaged to simplify perf testing
```

Comment:
Is this a good description? These are available for general use as far as I can tell, it doesn't only need to be perf testing.